internal/crawler/crawler.go

```diff
@@ -20,6 +21,7 @@ type Crawler struct {
 	gitlabClient *gitlab.Client
 	storage      storage.Storage
 	logger       zerolog.Logger
+	nWorkers     int
```

Comment:
We have a slight issue if the `gitlabCLient` is run with less RPS than we have workers, maybe that's worth catching and warning about 🤔